packages/gatsby-plugin-postcss-sass/src/gatsby-node.js

```diff
@@ -1,8 +1,11 @@
 import ExtractTextPlugin from "extract-text-webpack-plugin"
+import cssModulesConfig from "gatsby/dist/utils/css-modules-config"
```

Comment:
this is not a great pattern i don't think :/ it makes it really hard to understand what bits of the gatsby package are public and which aren't. Not to mention it ties the plugin to specific versions of gatsby that isn't documented anywhere

Reply:
Hmm yeah. We don't have good patterns yet for exposing public helper functions like this…

Reply:
OK. Would adding `gatsby` as a devDependency of these plugins be a bad idea? I guess so. Any suggestions for how to share the CSS Modules configuration between the core `webpack.config.js` and the different plugins? I thought about using an environment variable but didn't think that would be a good idea.

Reply:
Elaborating on this idea—I initially thought that storing the `localIdentName` config in an environment variable would not only make it reusable between plugins but also easily configurable by the user. However, the [documented method of setting environment variables via `.env.*` files](https://www.gatsbyjs.org/docs/environment-variables/) would not work because this environment variable would have to be available at the time `webpack.config.js` is evaluated, i.e. before `DefinePlugin` is loaded.

Reply:
I think v2 handles this neatly for the webpack case, we pass in essentially this helper to the plugin, vs the plugin requesting it.

Generally I think it makes the most sense to do either:
- pass what is needed to gatsby plugins via the plugin api
- extract the helper into a separately versioned package (MONOREEEEEEEPOOOOOO)

Reply:
> MONOREEEEEEEPOOOOOO

LOL! And yeah, that seems like the nicest solution for now — super easy to make a new package (`npm run plop`). Add `gatsby-1` to the package name so it's super obvious this is only needed for Gatsby 1 as it sounds like we can do things differently in v2.